src/RapidRequest.js

```diff
@@ -67,15 +67,19 @@ const processRequest = async (req) => {
   };
 };
 
-const executeRequest = async (request, locationDetails) => {
+const executeRequest = async (request) => {
   context = new Context({
-    ...request.testVariables,
-    ...request.envVariables,
+    ...(request.testVariables || {}),
```

Comment:
This function is used in the testing-service in the proxy handler. I wanted to make sure this function works if no envVariables or testVariables are passed.